src/libs/notifications.js

```diff
@@ -65,21 +65,16 @@ const NotificationDisplay = ({ id }) => {
   const onLast = notificationNumber + 1 === notifications.length;
 
   const { title, message, detail, type, action, onDismiss } = notifications[notificationNumber];
-  const [baseColor, ariaLabel] = Utils.switchCase(
+  const [baseColor, ariaLabel] = switchCase(
     type,
     ['success', () => [colors.success, 'success notification']],
     ['info', () => [colors.accent, 'info notification']],
     ['welcome', () => [colors.accent, 'welcome notification']],
     ['warn', () => [colors.warning, 'warning notification']],
     ['error', () => [colors.danger, 'error notification']],
-    [Utils.DEFAULT, () => [colors.accent, 'notification']]
```

Comment:
Looks like the default case got lost here...

Reply:
I think this one is still here. It's DEFAULT now instead of Utils.DEFAULT. I changed the import from src/libs to terra-ui-packages since src/libs was redirecting, and this let me get rid of the import of Utils.

Reply:
The diff confused me.